src/style.css

```diff
@@ -1 +1,13 @@
 /* styles go here */
+
+.fieldset {
+  margin-bottom: 20px;
+}
+
+.fieldset > div:not(:last-of-type) {
```

Comment:
Checklist point 6
It would be better to use another class (similar to 'comments-rec') and use it in divs around inputs and labels.